lib/assesments/assessment_2/fia2milyas/fia2milyas_view/fia2milyas_home/fia2milyas_homescreen.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/fia2milyas/fia2milyas_constant/fia2milyas_color.dart';
+import 'package:flutter_interns/assesments/assessment_2/fia2milyas/fia2milyas_view/fia2milyas_home/fia2milyas_custom_widget/fia2milyas_card.dart';
+import 'package:flutter_interns/assesments/assessment_2/fia2milyas/fia2milyas_view/fia2milyas_home/fia2milyas_custom_widget/fia2ilyas_homelistview.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+import 'fia2milyas_custom_widget/fia2milyas_home_menu.dart';
+
+class FiA2MIlyasHomeScreen extends StatelessWidget {
+  const FiA2MIlyasHomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    const lineheight = SizedBox(
+      height: 20,
+    );
+    return Scaffold(
+      bottomNavigationBar: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: ClipRRect(
```

Comment:
if you're using decoration it's better to add borderRadius in decoration than adding an extra widget